src/App.tsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const insensitiveQuery = query.toLowerCase().trim();
+  // const visibleMovies = [...moviesFromServer].filter(
+  //   movie => {
+  //     if (movie.title.toLowerCase().includes(insensitiveQuery)
+  //       || movie.description.toLowerCase().includes(insensitiveQuery)) {
+  //       return movie;
+  //     }
+
+  //     return null;
+  //   },
+  // );
+  const copmairingItems = (
+    item1: string,
```

Comment:
i think  isn't good name for variable and new lines here isn't needed

Reply:
agree